experiments/LES/dycoms.jl

```diff
@@ -0,0 +1,176 @@
+using StaticArrays
+using Test
+using CLIMA
+using CLIMA.Atmos
+using CLIMA.LowStorageRungeKuttaMethod
+using CLIMA.MoistThermodynamics
+using CLIMA.PlanetParameters
+using CLIMA.VariableTemplates
+
+const ArrayType = CLIMA.get_array_type()
+
+"""
+  Initial Condition for DYCOMS_RF01 LES
+@article{doi:10.1175/MWR2930.1,
+author = {Stevens, Bjorn and Moeng, Chin-Hoh and Ackerman,
+          Andrew S. and Bretherton, Christopher S. and Chlond,
+          Andreas and de Roode, Stephan and Edwards, James and Golaz,
+          Jean-Christophe and Jiang, Hongli and Khairoutdinov,
+          Marat and Kirkpatrick, Michael P. and Lewellen, David C. and Lock, Adrian and
+          Maeller, Frank and Stevens, David E. and Whelan, Eoin and Zhu, Ping},
+title = {Evaluation of Large-Eddy Simulations via Observations of Nocturnal Marine Stratocumulus},
+journal = {Monthly Weather Review},
+volume = {133},
+number = {6},
+pages = {1443-1462},
+year = {2005},
+doi = {10.1175/MWR2930.1},
+URL = {https://doi.org/10.1175/MWR2930.1},
+eprint = {https://doi.org/10.1175/MWR2930.1}
+}
+"""
+function init_dycoms!(state::Vars, aux::Vars, (x,y,z), t)
+    FT            = eltype(state)
+
+    xvert::FT     = z
+    Rd::FT        = R_d
+
+    # These constants are those used by Stevens et al. (2005)
+    qref::FT      = FT(9.0e-3)
+    q_tot_sfc::FT = qref
+    q_pt_sfc      = PhasePartition(q_tot_sfc)
+    Rm_sfc::FT    = 461.5 #gas_constant_air(q_pt_sfc)
```

Comment:
I think it's better if we use our own thermodynamic constants consistently. Otherwise, we are bound to get inconsistencies in the thermodynamics. (Once we have a way of overriding `PlanetParameters` for the entire code, we can define local thermodynamic constants for specific cases.)